frontend/src/components/ExpandableColumn.tsx

```diff
@@ -12,26 +12,38 @@ export const ExpandableColumn: FC<{
   title: ReactNode;
   className?: string;
   style?: CSSProperties;
-}> = ({ expanded, onToggle, title, children, className, style }) => (
-  <div className={className} style={style}>
-    <div
-      className={classes(css.expandableContentWrapper, {
-        [css.minimized]: !expanded,
-      })}
-    >
+}> = ({ expanded, onToggle, title, children, className, style }) => {
+  // delay showing children, to avoid breaking possible measuring divs
+  const [showChildren, setShowChildren] = useState(expanded);
+
+  useEffect(() => {
+    if (!expanded) setShowChildren(false);
+    else setTimeout(() => setShowChildren(true), 300);
```

Comment:
```suggestion
    if (!expanded) {
      setShowChildren(false);
    } else {
      const t = setTimeout(() => setShowChildren(true), 300);
      return () => clearTimeout(t);
    }
```

If toggling quickly expanded -> closed, the timeout will stay alive and set the children to be rendered when the column is already closed.

Also it would be good to cancel the timeout. I'm not 100% sure if this on it's own is enough for the above bug though.